docs/community-ee/community-ee-release-process.md

```diff
@@ -11,68 +11,65 @@ The EE versioning convention is core tag plus patch number, for example:
     - EE with core `2.16.2` comes out -> `community-ee:2.16.2-1`
     - EE with core `2.16.3` comes out -> `community-ee:2.16.3-1`
 
-## Dependencies
-
-- Podman/Docker
-
 ## Credentials
 
-- Access to the https://github.com/ansible-community/images repository
-- Access to ghcr.io
+- Access to the https://github.com/ansible-community/images repository.
+- Join the [Release Management working Matrix room](https://app.element.io/?updated=1.11.38#/room/#release-management:ansible.com) and the [Execution Envs group in Forum Group](https://forum.ansible.com/g/ExecutionEnvs).
+- Access to Ansible Release Management Group in Github.
+- Access to the [eercheck](https://github.com/anweshadas/eercheck) repository.
 
 ## Prerequisites
 
 - Join the [Release Management working group](https://forum.ansible.com/g/release-managers) and [Execution Environment group](https://forum.ansible.com/g/ExecutionEnvs).
 - Understand [Ansible execution environments](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341).
 - Read about the [eerelease.yml](/.github/workflows/eerelease.yml)  GitHub workflow.
 - Show intention and book the date for the releasing of the EE.
+- Prepare the release announcement drafts for the emails and matrix communication.
+- Shadow the release manager before doing the actual release.
 
-## Making the pre-release announcement
+## Build Steps
 
-Post this message in the channel before working on the release, using the correct release number.
+Open a new terminal window and then complete the following steps:
 
-```markdown
-📯📯📯 I am working on the `Ansible Execution Environment Base and Minimal 2.x.y-1` Release. I will keep the room updated about the progress.📦️
-```
+1. Check the ansible-core version [here](https://pypi.org/project/ansible-core/)
+2. Verify ansible collection versions for `ansible.posix`, `ansible.utils` and  `ansible.windows` for `deps` file of the ansible version for the related release  [ansible-build-data repo](https://github.com/ansible-community/ansible-build-data).
+3. Change to the `images/execution-environments` directory.
+4. Create a git branch following the naming convention of `ansible-core-<ee_version>`, where <ee_version> matches the related version; for example, 2.17.1-1.
 
-## Building the EEs
+```
+	git checkout main
+	git pull upstream main
+	git checkout -b core_version-ee_version
+```
```

Comment:
In this codeblock and the next one, you should reduce some of the whitespace. You probably also want to indent the three ticks that start and close the codeblock so it is nested within the step.

Currently the codeblocks look like this:

![image](https://github.com/user-attachments/assets/ae4ac3ee-5b79-4ab3-bfdf-bf9b86f6d1db)